tests/python/unittest/test_sparse_operator.py

```diff
@@ -314,7 +314,7 @@ def check_softmax_with_shape(lhs_stype, rhs_stype, shape, preserve_shape=False):
         out = exec1.outputs[0].asnumpy()
         assert_almost_equal(out, np_softmax(x.asnumpy()), rtol=1e-4)
         exec1.backward()
-        assert_almost_equal(grad.asnumpy(), np_softmax(x.asnumpy()) - l.asnumpy(), rtol=1e-4)
+        assert_almost_equal(grad.asnumpy(), np_softmax(x.asnumpy()) - l.asnumpy(), rtol=1e-3, atol=1e-4)
```

Comment:
It passed. But I still broke it into two lines so that there are less than 100 characters each line.